setup.py

```diff
@@ -25,7 +25,7 @@
 '''
 
 setup(name='keras-mxnet',
-      version='2.2.0',
+      version='2.2.2',
```

Comment:
Please update version for `keras_applications` & `keras_preprocessing` 
See https://github.com/keras-team/keras/blob/master/setup.py#L40